samples/drivers/jesd216/sample.yaml

```diff
@@ -4,6 +4,7 @@ tests:
   sample.drivers.jesd216:
     tags: spi flash
     filter: dt_compat_enabled("jedec,spi-nor")
+    platform_exclude: mimxrt1170_evk_cm7 mimxrt1170_evk_cm4
```

Comment:
Hmm, is there a reason we need this.  Can we not enable CONFIG_MEMC_MCUX_FLEXSPI for rt1170?

Reply:
hi， 
    in fact, the CONFIG_MEMC_MCUX_FLEXSPI is not enabled, so there is a link error, said  **undefined reference**. and also the rt1170's flash is with the "jedec,spi-nor" compatible: **compatible = "issi,is25wp128", "jedec,spi-nor";** so this test will run for the rt1170, but without include the SPI driver(which in rt1170 is flexspi, not spi instead). so i think that we
can use the platform_exclude to filter the rt1170_evk board.


Regards,
   Crist